cypress/e2e/spec.cy.js

```diff
@@ -76,32 +76,46 @@ describe('Register Tests', () => {
     cy.get('[placeholder="Email"]').type('email@email.com')
     cy.get('[placeholder="Password"]').type('password')
 
-    cy.intercept(`*register`, {
+    cy.intercept(`*user`, {
       statusCode: 403,
     })
 
-    cy.contains("Let's go!").click()
-
-    cy.on('windows:alert', (str) => {
-      expect(str).to.equal('Username or Email already taken, or invalid inputs')
-    })
+    const stub = cy.stub()
+    cy.on('window:alert', stub)
+    cy.contains("Let's go!")
+      .click()
+      .then(() => {
+        expect(stub.getCall(0)).to.be.calledWith(
+          'Username or Email already taken, or invalid inputs'
+        )
+      })
   })
 })
 
 describe('Login Tests', () => {
   it('should login', () => {
     cy.visit(Cypress.env('BASE_URL'))
 
+    cy.contains('UnReveal').should('be.visible')
+    cy.contains('Login').should('be.visible')
+    cy.contains('Register').should('be.visible')
+
     cy.contains('Login').click()
 
     cy.get('[placeholder="Email"]').type('email@email.com')
     cy.get('[placeholder="Password"]').type('password')
 
     cy.contains("Let's go!").click()
 
-    cy.on('windows:alert', (str) => {
-      expect(str).to.equal('Account created!')
-    })
+    const stub = cy.stub()
+    cy.on('window:alert', stub)
+    cy.contains("Let's go!")
+      .click()
+      .then(() => {
```

Comment:
This test should login successfully. If it displays user not found the it is not logging in